src/bind.c

```diff
@@ -285,7 +285,15 @@ void df_c_fallback(r_obj* out,
       FREE(1);
     } else if (vec_is_common_class_fallback(ptype_col)) {
       r_obj* xs_col = KEEP(list_pluck(xs, i));
-      r_obj* out_col = vec_c_fallback(ptype_col, xs_col, name_spec, name_repair, error_call);
+
+      r_obj* out_col = vec_c_fallback(
```

Comment:
By the way AFAIK Emacs can't indent that and will fight against this type of indentation. I'm fine with it but we used indent-at-paren until recently. If we generally switch to that style though, I'm going to have a hard time editing vctrs sources :-(

Maybe that has changed since I looked a few years ago. Do you know if this style of indentation has a known name?

Reply:
It matches the R level tidyverse style guide https://style.tidyverse.org/syntax.html#long-lines and is what I find most readable 😢 , I like that it moves everything closer to the left hand side so I can read top to bottom rather than having to move my eyes left->right and back so much.

If it is not possible in Emacs we can use indent-at-paren (I actually don't think I ever used that 😬 ) but it would be great to try to see if Emacs can do this!